spec/lib/rufo/formatter_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+
+RSpec.describe Rufo::Formatter do
```

Comment:
I'd actually move all the other `rufo_spec` tests into this file, since really those tests are testing the formatter. Would you mind doing that after I merge my PR?